### PR TITLE
Add Blogody to No-Code Platforms section

### PR DESCRIPTION
**Pull request by styxlab**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,6 +77,7 @@
 *For more resources about Static Web Apps see (Awesome Static Web Apps)[https://github.com/staticwebdev/awesome-azure-static-web-apps].*
 
 ## No-Code Platforms
+- [Blogody](https://www.blogody.com) - Create your own blog and see it live in seconds.
```
Comment (automata) [inline README.md:80]: Should be placed at end of list of items.

**Review by automata — CHANGES_REQUESTED** [review]

**Comment by styxlab** [comment]
Also added it to the headless CMS section as Blogody can be used both in "no-code" and "headless" mode.

**Comment by automata** [comment]
@styxlab please add in only one section and fix the conflict.

**Comment by automata** [comment]
Closing for now for the lack of updates, @styxlab please feel free to re-open it.

**Pull request closed without merging**